### PR TITLE
REM-454

### PR DESCRIPTION
**Pull request by imtithalhalalo**
new page: “sc-end-site-count”

### CODE REVIEW

**Comment by FatimaSalami** [comment]
there is a problem in behaviour, when you make certain selections and post..
if you select them again the icon should be locked not as if it is first time to select them

File: src/pages/sc-end-site-count/forms/EndSiteCountForm.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    {
+      key: 'Lock',
+      condition: formik.values.statusName === 'Processed',
+      onClick: () => formik.handleSubmit(),
+      disabled: false
+    },
+    {
+      key: 'Unlock',
+      condition: formik.values.statusName !== 'Processed',
+      onClick: () => formik.handleSubmit(),
```
Comment (Alinadimmokdad) [inline src/pages/sc-end-site-count/forms/EndSiteCountForm.js:69]: is it better to use status =1 or 2 ?

File: src/pages/sc-end-site-count/forms/EndSiteCountForm.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                maxAccess={access}
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <CustomTextField
+                name='notes'
+                type='text'
+                label={_labels.notes}
+                readOnly
+                value={formik.values.notes}
```
Comment (Alinadimmokdad) [inline src/pages/sc-end-site-count/forms/EndSiteCountForm.js:127]: is it readOnly ? 
if no use customTextArea

**Review by Alinadimmokdad — APPROVED** [review]

File: src/pages/sc-end-site-count/forms/EndSiteCountForm.js (hunk 2 above)
Comment (imtithalhalalo) [reply src/pages/sc-end-site-count/forms/EndSiteCountForm.js:127]: yes readOnly

File: src/pages/sc-end-site-count/forms/EndSiteCountForm.js (hunk 1 above)
Comment (imtithalhalalo) [reply src/pages/sc-end-site-count/forms/EndSiteCountForm.js:69]: same

Comment (NoraBitarSM) [reply src/pages/sc-end-site-count/forms/EndSiteCountForm.js:69]: @imtithalhalalo don't compare based on the statusName, what if we're using argus in arabic ? @Alinadimmokdad is right

**Pull request merged**